salt/minion.py

```diff
@@ -1881,6 +1881,9 @@ def handle_schedule():
     def _handle_payload(self, payload):
         if payload is not None and self._target_load(payload['load']):
             self._handle_decoded_payload(payload['load'])
+        elif self.opts['zmq_filtering']:
```

Comment:
Good point. I'd like to leave as debug if that is alright (trace is a bit chatty). I used this line to verify that minions I was not targeting where not getting the payload. On that note though, what about:
- Leaving it as debug
- elif  self.opts['zmq_filtering'] and not self._target_load(payload['load']):
- and adding the payload to the log line?